source/includes/_lips_contacts.md

```diff
@@ -11,6 +11,7 @@
 | `create` | POST | /contacts |
 | `update` | PUT | /contact/[id] |
 | `destroy` | DELETE | /contact/[id] |
+| `show` | GET | /contacts/[id]/events |
```

Comment:
Our documentation structure makes this hard, but this isn't a Contacts API.  It's an Events API with a clever REST filter.  It's not the Contact `show` route.  Not sure what to put there instead.